src/dotnet/scripts/dotnet-helpers.sh

```diff
@@ -76,12 +75,11 @@ install_sdk() {
     fi
     
     # Currently this script does not make it possible to qualify the version, 'GA' is always implied
-    echo "Executing $DOTNET_INSTALL_SCRIPT --version $version --channel $channel --install-dir $DOTNET_INSTALL_DIR --no-path"
```

Comment:
Curious, why was `--no-path` removed?

Reply:
The default behavior is to add `dotnet` to the PATH for the current shell session. That didn't seem very useful to me, so I initially added `--no-path` to disable modifying the PATH.

Now, I do need to run `dotnet` in the same shell session (to install workloads), so I re-enabled modifying PATH.